react/features/base/toolbox/components/web/ToolboxButtonWithPopup.tsx

```diff
@@ -142,6 +142,7 @@ export default function ToolboxButtonWithPopup(props: IProps) {
             <div className = 'settings-button-small-icon-container'>
                 <Popover
                     content = { popoverContent }
+                    focusable = { false }
```

Comment:
Oh boy I must be asleep because I don't understand what you mean :upside_down_face: 

The reactions menu and audio/video menus don't behave the same for now as the reactions menu trigger on hover and audio/video menus trigger on click.

Reply:
Aaa yes. I didn't catch that, it's true, reactions menu appears when you hover. I thought the focus didn't get lock inside the reactions menu when one navigates using tab key, but it does.

Reply:
Wait. I tested on production and, when pressing tab, after Raised hand button get focused the next focusable element is the Reactions menu button and after that the Reactions inside the Reactions menu. With your new updates, this doesn't happen anymore. From focusing the Raised hand button, the next focused element is Participants button. So I think the approach that we have on production is the one that we want from a ux perspective.

Reply:
Oh good catch, this is not intentional indeed, must have forgot something when rebasing, will change!

Reply:
I just removed the `focusable = { false}` prop so that it works correctly. Forgot to change this when removed the duplicate reactions menu button.

Reply:
It works, but I think you forgot to remove the prop. 
At least I had to remove it when I checked out your branch to test the changes.

Reply:
Seems ok to me: https://github.com/Leimi/jitsi-meet/blob/fix-a11y-missing-labels/react/features/base/toolbox/components/web/ToolboxButtonWithPopup.tsx

I did update by editing an existing commit instead of making a new one so maybe that's why.